app/models/index_queue.rb

```diff
@@ -22,8 +22,8 @@ def url
   end
 
   def parsed_response
-    JSON.parse(response).first['datapoints'].first.first
-  rescue JSON::ParserError, TypeError
+    JSON.parse(response.to_s).first['datapoints'].first.first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.